pos_customer_required/tests/test_pos.py

```diff
@@ -0,0 +1,49 @@
+import odoo.tests.common as common
+from odoo import exceptions
+
+
+class TestPosRequireCustomer(common.TransactionCase):
+    def setUp(self):
+        super(TestPosRequireCustomer, self).setUp()
+        self.pos_config = self.env.ref("point_of_sale.pos_config_main").copy()
```

Comment:
```suggestion
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        self.pos_config = self.env.ref("point_of_sale.pos_config_main").copy()
```

`setUp` runs for every case
`setUpClass` runs only once for all

Reply:
mmmhhhh!!! This is breaking the test case. Should I revert?

Reply:
I did!

Reply:
I had a typo in my suggestion
It should've been like this

```suggestion
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        cls.pos_config = self.env.ref("point_of_sale.pos_config_main").copy()
```

Anyway feel free to keep it as it is

Reply:
Let's keep as before because using ``cls`` instead of ``self`` doesn't seem to work, in my patch I've replace ``self -> cls`` and indeed ``self`` isn't available.